elements/map/src/components/tooltip.js

```diff
@@ -8,6 +8,7 @@ export class EOxMapTooltip extends LitElement {
   static get properties() {
     return {
       feature: { attribute: false, property: true, type: Object },
+      for: { attribute: "for", property: true, type: String },
```

Comment:
Usually `for` is used for DOM specifiers. Maybe we should use a different property for the layer ID.